src/libraries/Native/AnyOS/brotli/dec/decode.c

```diff
@@ -24,6 +24,16 @@
 #include <arm_neon.h>
 #endif
 
+#ifndef __has_attribute
```

Comment:
We should disable warning for brotli instead. We typically avoid fixing warnings in the vendored code.

Reply:
Makes sense.